frontend/src/components/RecipeSteps.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components/macro'
+
+export default function RecipeSteps({steps}) {
+    return (
+        <StepsWrapper>
+            <p>Zubereitung</p>
+            <ol>
+                {steps.sort((oneStep, anotherStep) => oneStep.stepNumber.localeCompare(anotherStep.stepNumber))
```

Comment:
Careful, `sort` will modify the array in place, maybe make a quick copy with `.slice()` before, as we don't want to change props (`steps`) directly.